### PR TITLE
Feat (Ai/all): basic client, agent, repetoile

### PR DESCRIPTION
**Pull request by RoheeAxel**
This PR does the following:

• added basic python client to connect to the server
• added class Agent who is basic agent with funtion to call server
• added class Repetoile who is basic agent who can survive

The code has been tested and no problems were found on my side.

### CODE REVIEW

**Review by Sinan-Karakaya — APPROVED** [review]
seems good

File: zappy_ai/main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
```
Comment (JasonLAGOUTE) [inline zappy_ai/main.py:15]: Header tek ?

**Review by JasonLAGOUTE — COMMENTED** [review]
code seems good

**Review by noerio — APPROVED** [review]
code is good, well documented

**Pull request merged**